src/visidentifier/vismessage.hpp

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef VISMESSAGE_HPP_
+#define VISMESSAGE_HPP_
+
+#include <array>
+#include <string>
+
+#include <Poco/JSON/Object.h>
+
+#include <aos/common/tools/array.hpp>
+#include <aos/common/tools/enum.hpp>
+
+/***********************************************************************************************************************
+ * Static
+ **********************************************************************************************************************/
+
+static constexpr const char* cActionTagName         = "action";
```

Comment:
i think this static variables should move to private section on class VISMessage

Reply:
The idea was to use this constants for unit test as well (instead of magic strings). 
But I can refactor if you insist

Reply:
moved to public section of VISMessage class.
These constants are used in other cpp(s)